src/pytest_inmanta_lsm/orchestrator_container.py

```diff
@@ -44,8 +44,6 @@ def run_cmd(*, cmd: List[str], cwd: Path) -> Tuple[str, str]:
             env=env_vars,
         )
     except FileNotFoundError as e:
-        if e.filename == "docker-compose":
-            raise FileNotFoundError("The `docker-compose` command is not found. You need it to have a local orchestrator.")
```

Comment:
If this goes away, then catching the exception doesn't make sense anymore

Reply:
Do you think it still makes sense keeping this check here? Is there another use for run_cmd besides the ones that I changed?